db/data/regions/BR.yml

```diff
@@ -19,6 +19,19 @@ languages:
 format:
   edit: "{country}_{firstName}{lastName}_{company}_{zip}_{address1}_{address2}_{city}{province}_{phone}"
   show: "{firstName} {lastName}_{company}_{address1}_{address2}_{zip} {city} {province}_{country}_{phone}"
+additional_address_fields:
+  address1:
+    - key: streetName
+      required: true
+    - key: streetNumber
+      decorator: ","
+      required: true
```

Comment:
Requiring street number on BR checkouts is a new requirement. How is this signaled or enforced by servers & frontends?